.github/workflows/main.yml

```diff
@@ -48,8 +50,8 @@ jobs:
           AUTH0_CLIENT_SECRET: ${{ secrets.AUTH0_CLIENT_SECRET }}
 
       - name: Update codecov report
-        uses: codecov/codecov-action@v2
+        uses: codecov/codecov-action@v3
         with:
-          file: coverage.out
-          fail_ci_if_error: true
+          files: ./coverage.out
+          fail_ci_if_error: false
```

Comment:
We don't want the entire workflow to fail if we fail to upload the coverage report.

Reply:
Is this masking some underlying issue? Not that we should be disrupting the entire workflow if codecov fails, but is this actually a fix?

Looking at the error below, it looks like we're not passing the token along to the server, doesn't strike me as being an intermittent issue.

```
There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 404 - Not Found - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
```


Looks like some other folks have this issue, it's still marked as open but perhaps some solutions in here: https://github.com/codecov/codecov-action/issues/598.

Reply:
Code coverage is a non essential task in our tests workflow, if it fails the entire build should not fail. 

Also the token is not required for public repos. 

The action fails randomly at times unfortunately. After a few retries it eventually was able to push the coverage correctly.